src/components/ui/Pressable/Pressable.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import Pressable from './Pressable';
+
+export default {
+  title: 'Ui/Pressable',
+  component: Pressable,
+  argTypes: {},
+} as ComponentMeta<typeof Pressable>;
+
+const Template: ComponentStory<typeof Pressable> = (args) => (
+  <Pressable {...args}>
+    <p>Click me</p>
+  </Pressable>
+);
+
+export const Normal = Template.bind({});
+Normal.args = {};
+
+export const H3 = Template.bind({});
+H3.args = {
+  children: <h3>Click me</h3>,
```

Comment:
`children` here doesn't affect the actual representation of the component
You can see it if you will remove `p` from the `Template`. h3/h5 stories will show black text without underline
Let's figure out what is the issue